docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+---
+lang: en
+title: 'Deploying to Cloud with Appsody'
+keywords: LoopBack 4.0, LoopBack 4, Appsody
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Appsody-LoopBack.html
+---
+
+## LoopBack 4 & Appsody
```

Comment:
The header can be `## Introduction` or `## Overview`.

Reply:
Choosing Overview.